setup.cfg

```diff
@@ -0,0 +1,42 @@
+[bumpversion]
+current_version = 0.0.1
+commit = True
+tag = True
+
+[bumpversion:file:docs/conf.py]
+search = release = "{current_version}"
+replace = release = "{new_version}"
+
+[bumpversion:file:setup.py]
+search = version="{current_version}"
+replace = version="{new_version}"
+
+[bumpversion:file:craft_parts/__init__.py]
+search = __version__ = "{current_version}"
+replace = __version__ = "{new_version}"
+
+[bdist_wheel]
+universal = 1
+
+[codespell]
+quiet-level = 3
+skip = .mypy_cache,.direnv,.git
+
+[flake8]
+exclude = docs venv .venv .mypy_cache .direnv .git __pycache__
+max-line-length = 88
+extend-ignore = E203, E501, W503
+
+[mypy]
+python_version = 3.8
+
+[pycodestyle]
+ignore = E402, E501, W503
```

Comment:
fwiw, in craft-providers I've been trying to maintain the list of explanations alongside the codes. e.g.

```
[pydocstyle]
# D105 Missing docstring in magic method (reason: magic methods already have definitions)
# D107 Missing docstring in __init__ (reason: documented in class docstring)
# D203 1 blank line required before class docstring (reason: pep257 default)
# D213 Multi-line docstring summary should start at the second line (reason: pep257 default)
# D215 Section underline is over-indented (reason: pep257 default)
ignore = D105, D107, D203, D213, D215
```

Reply:
Resynced setup.cfg against craft-providers as well.